dashboard/src/components/Catalog/Catalog.test.tsx

```diff
@@ -88,7 +88,7 @@ const defaultState = {
   charts: defaultChartState,
   operators: { csvs: [] } as Partial<IOperatorsState>,
   repos: { repos: [] } as Partial<IAppRepositoryState>,
-  config: { kubeappsCluster: "default", kubeappsNamespace: "kubeapps" } as IConfigState,
+  config: { kubeappsCluster: defaultProps.cluster, kubeappsNamespace: defaultProps.kubeappsNamespace } as IConfigState,
```

Comment:
Tests were depending on "default-cluster", but getting "default" because of this config difference.